inspirehep/modules/workflows/tasks/arxiv.py

```diff
@@ -122,7 +122,7 @@ def arxiv_plot_extract(obj, eng):
             return
 
         for idx, plot in enumerate(plots):
-            obj.files[plot.get('name')] = BytesIO(open(plot.get('url')))
```

Comment:
@jacquerie why you say so? Only on jython. The file is closed as soon as it gets out of scope, AFAIK